js/signin.js

```diff
@@ -1,114 +1,82 @@
-const QUERY_SELECTOR = function (select) {
-  return document.querySelector(select);
-};
+import { $, SIGN_INFO_REGEX } from './common.js';
+import {
+  setEmailErrorStyle,
+  resetEmailStyle,
+  setPasswordErrorStyle,
+  resetPasswordStyle,
+  styleLoginError,
+} from './signErrorStyle.js';
+import { hidePassword, showPassword } from './passwordShowStyle.js';
 
-const $emailInput = QUERY_SELECTOR('#emailInput');
-const $emptyEmailError = QUERY_SELECTOR('#emptyEmailError');
-const $wrongEmailError = QUERY_SELECTOR('#wrongEmailError');
+const $jsEmailInput = $('.js-email-input');
+const $jsPasswordInput = $('.js-password-input');
 
-$emailInput.addEventListener('focusout', function (e) {
-  const emailRegEx = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-Za-z0-9\-]+/;
+$jsEmailInput.addEventListener('blur', (e) => {
+  resetEmailStyle();
 
-  defaultStyle();
+  if (e.target.value === '') {
+    setEmailErrorStyle('이메일을 입력해주세요.');
+    return;
+  }
 
-  if (this.value === '') {
-    styleSeting('email', 'empty');
-  } else if (!emailRegEx.test(this.value)) {
-    styleSeting('email', 'wrong');
+  if (!SIGN_INFO_REGEX.email.test(e.target.value)) {
+    setEmailErrorStyle('올바른 이메일 주소가 아닙니다.');
+    return;
   }
 });
 
-const $passwordInput = QUERY_SELECTOR('#passwordInput');
-const $emptyPasswordError = QUERY_SELECTOR('#emptyPasswordError');
+$jsPasswordInput.addEventListener('blur', (e) => {
+  resetPasswordStyle();
 
-$passwordInput.addEventListener('focusout', function (e) {
-  defaultStyle();
+  if (e.target.value === '') {
+    setPasswordErrorStyle('비밀번호를 입력해주세요.');
+    return;
+  }
 
-  if (this.value === '') {
-    styleSeting('password');
+  if (!SIGN_INFO_REGEX.password.test(e.target.value)) {
+    setPasswordErrorStyle('비밀번호는 영문, 숫자 조합 8자 이상 입력해 주세요.');
   }
 });
 
-const $loginButton = QUERY_SELECTOR('#btnLogin');
-const $checkEmailError = QUERY_SELECTOR('#checkEmailError');
-const $checkPasswordError = QUERY_SELECTOR('#checkPasswordError');
+const ENTER_KEY = 'Enter';
 
-function loginBtnClick() {
-  defaultStyle();
+$jsEmailInput.addEventListener('keydown', (e) => {
+  e.key === ENTER_KEY && $jsPasswordInput.focus();
+});
```

Comment:
취향으로 볼 수 있기는 하지만 if문으로 처리해주는게 더 깔끔할 것 같아요!
```suggestion
$jsEmailInput.addEventListener('keydown', (e) => {
  if(e.key === ENTER_KEY) {
    $jsPasswordInput.focus();
  }
});
```